keyboards/dz60/keymaps/olivierko/keymap.c

```diff
@@ -0,0 +1,87 @@
+#include QMK_KEYBOARD_H
+
+enum keyboard_layers {
+  _BL = 0,
+  _SL = 1,
+  _FL = 2,
+  _CL = 3,
+};
```

Comment:
You actually don't need any of the `= #` here, `enum` automatically starts at index 0 and increments by 1 (unless you explicitly tell it to do otherwise).

```
enum keyboard_layers {
  _BL,
  _SL,
  _FL,
  _CL,
};
```